### DISCUSSION TITLE
Apple homekit app

### DISCUSSION THREAD

**Discussion by andrealm10** [RavenSystem/esp-homekit-devices#2268] [HAA Home App]
Hello, this is a problem I have had for a very long time. When I add a relay I can set up the Wifi and connect to the manger. But when I go to the Homekit app on iPhone and try to add a new device it never shows up! I have tried many different relays and Wifi boards. I had a break for 6 month because it was so annoying and now I was gonna try again. 
![Screenshot 2024-06-14 at 21 54 43](https://github.com/Raven
![IMG_2265](https://github.com/RavenSystem/esp-homekit-devices/assets/141873518/2ff1d4ae-d066-4b0d-a2b0-508e31d92544)
System/esp-homekit-devices/assets/141873518/182814ba-0859-4e5d-8608-b22e713ad478)

**Comment by andrealm10** [comment]
![Screenshot 2024-06-14 at 21 54 43](https://github.com/RavenSystem/esp-homekit-devices/assets/141873518/a9aa9d4f-64a6-423e-90fa-a6ff5b3076c6)

**Comment by andrealm10** [comment]
![Screenshot 2024-06-14 at 22 34 26](https://github.com/RavenSystem/esp-homekit-devices/assets/141873518/b0be578d-312e-458a-a387-25b066ec795c)

**Reply by tsgxwolf** [reply]
Seems you did not connect the WIFI.Try to Enter the wifi ssid manually.And do not choose “Reset the homekit ID”
![IMG_4749](https://github.com/RavenSystem/esp-homekit-devices/assets/47149325/d1ed90a0-2232-481b-846d-68c225955208)

**Reply by andrealm10** [reply]
Hello, thank you for the reply. 
This picture just didn't show with when Wifi was entered. I have tried many many times with wifi and “Reset the homekit ID” not selected. The problem is that the device will not show up in the Apple Home app. I don't know what is blocking it and/or making it never showing up. 
// André

**Reply by tsgxwolf** [reply]
so,if you connected a 2.4g wifi,and the dhcp setting is correct,then you need to confirm if the MEPLHAA Script is correct.Your pic shows the device name HAA-163A97-setup，it usualy happens when the script is not working,and the device will enter the setup mode

**Comment by shengweitw** [comment]
If it is a new device, please wait a little longer, sometimes 10 minutes, but it may also take half an hour. If it is an old setting, you must Reset Homekit ID.
